chia/cmds/plotnft.py

```diff
@@ -103,7 +105,15 @@ def create_cmd(
 
 @plotnft_cmd.command("join", short_help="Join a plot NFT to a Pool")
 @click.option("-y", "--yes", help="No prompts", is_flag=True)
-@click.option("-i", "--id", help="ID of the wallet to use", type=int, default=None, show_default=True, required=True)
+@click.option(
+    "-i",
+    "--id",
+    help="ID of the Pooling wallet to store pool rewards",
```

Comment:
It feels to me like the primary consideration would be that the user is specifying the pool wallet associated with the plots they want farming for the pool.  The rewards going to that wallet seems more like an effect than a goal.